home.html

```diff
@@ -18,128 +19,274 @@
-	<a href="rasgulla.html"><h1 class="title">Rasgulla</h1></a>
-		<img src="image/rasgulla.jpg" class="img-responsive">
-	</div>
-	
-
-
-
-
-
-
-
-
-
-
-
+  </div>
+</br>
+</br> 
+  <div class="container">
+  <div class = "row">
+  </div>
+  <div class="row">
+  <div class="col-md-4">
+    
+  </div>
+  <div class="col-md-4">
+    <h1 class="title">More books</h1> 
+       
+  </div>
+  <div class="col-md-4">
+      
+  </div>
+  <div class = "row">
+
+  </div>
+  </div>
+
+</br>
+</br> 
+  <div class="row">
+      <div class="col-md-4">
+  <div class="card" style="width: 22rem; margin: 6px">
+    <img class="card-img-top" src="https://images-na.ssl-images-amazon.com/images/I/41fRAMCMu1L._SX258_BO1,204,203,200_.jpg" alt="Card image cap">
+    <div class="card-body">
+      <h5 class="card-title">Algorithm</h5>
+      <p class="card-text">This is algorithm book</p>
+      <a href="#" class="btn btn-primary">Go somewhere</a>
+    </div>
+  </div>
+</div>
+<div class="col-md-4">
+  <div class="card" style="width: 22rem; margin: 6px">
+    <img class="card-img-top" src="https://images-na.ssl-images-amazon.com/images/I/51dYmGrWCOL._SX258_BO1,204,203,200_.jpg" alt="Card image cap">
+    <div class="card-body">
+      <h5 class="card-title">Card title</h5>
+      <p class="card-text">This is algorithm book</p>
+      <a href="#" class="btn btn-primary">Go somewhere</a>
+    </div>
+  </div>
+</div>
+<div class="col-md-4">
+    <div class="card" style="width: 22rem; margin: 6px">
+      <img class="card-img-top" src="https://images-na.ssl-images-amazon.com/images/I/51dYmGrWCOL._SX258_BO1,204,203,200_.jpg" alt="Card image cap">
+      <div class="card-body">
+        <h5 class="card-title">Card title</h5>
+        <p class="card-text">This is algorithm book</p>
+        <a href="#" class="btn btn-primary">Go somewhere</a>
+      </div>
+    </div>
+  </div>
+</div>
+  <div class="row">
+      <div class="row">
+          <div class="col-md-4">
+      <div class="card" style="width: 22rem; margin: 6px">
+        <img class="card-img-top" src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTcjLxDSp29Z8GD1FyOUcYwuJK4ZCbPARfukzxG6GfqtQ54Gzi8kBarys2TAdQ&usqp=CAc" alt="Card image cap">
+        <div class="card-body">
+          <h5 class="card-title">Algorithm</h5>
+          <p class="card-text">This is algorithm book</p>
+          <a href="#" class="btn btn-primary">Go somewhere</a>
+        </div>
+      </div>
+    </div>
+    <div class="col-md-4">
+      <div class="card" style="width: 22rem; margin: 6px">
+        <img class="card-img-top" src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTcjLxDSp29Z8GD1FyOUcYwuJK4ZCbPARfukzxG6GfqtQ54Gzi8kBarys2TAdQ&usqp=CAc" alt="Card image cap">
+        <div class="card-body">
+          <h5 class="card-title">Card title</h5>
+          <p class="card-text">This is algorithm book</p>
+          <a href="#" class="btn btn-primary">Go somewhere</a>
+        </div>
+      </div>
+    </div>
```

Comment:
- All of cards should have the same height and width. Some of them are longer than other.
- Replace this section "This is algorithm book"  with book description and uses css to trim overflow text.